app/src/pages/inside/stepPage/stepPage.jsx

```diff
@@ -457,7 +458,7 @@ export class StepPage extends Component {
             listView={listView}
             onEditDefect={this.handleEditDefects}
             onUnlinkSingleTicket={this.handleUnlinkSingleTicket}
-            events={STEP_PAGE_EVENTS}
+            events={{ STEP_PAGE_EVENTS, getOnClickIssueTicket }}
```

Comment:
I guess it break all events inside StepGrid. Maybe STEP_PAGE_EVENTS should be spread(...)